blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -144,6 +144,10 @@ Deprecations
 Changes
 =======
 
+- Add `minimum_index_compatibility_version` and
+  `minimum_wire_compatibility_version` to sys.node table to expose current
```

Comment:
please link to the relevant sections in the docs.. (which should be updated, see https://github.com/crate/crate/blob/master/blackbox/docs/admin/system-information.rst#nodes)